cypress/support/commands.js

```diff
@@ -0,0 +1,149 @@
+    cy.fixture(filePath).then(options => {
+        const selector = `${Selectors.selectedOptionsTitle1}:contains(${options.make}), ${Selectors.selectedOptionsTitle2}:contains(${options.make})`
+        cy.get(selector).should('exist').and('be.visible')
+    })
+})
+
+Cypress.Commands.add('selectBasicTrim', () => {
+    cy.get(Selectors.trimOptions).children().eq(0).find('a').click()
+})
+
+Cypress.Commands.add('storeCarCharacteristics', (fileName) => {
+    const carInfo = {}
+    cy.get(Selectors.carCharacteristics).contains(testData.doorCharacteristic).then(($element) => {
+        carInfo.door = $element.text().trim()
+    })
+    cy.get(Selectors.carCharacteristics).contains(testData.seatCharacteristic).then(($element) => {
+        carInfo.seat = $element.text().trim()
+    })
+    cy.get(Selectors.carCharacteristics).contains(/Engine|Motor/i).then(($element) => {
+        carInfo.engine = $element.text().trim()
+    }).then(() => {
+        const filePath = testData.files + fileName
+        cy.writeFile(filePath, carInfo)
+    })
+})
+
+Cypress.Commands.add('goToMainPage', () => {
+    cy.visit(testData.carsMainPage, { headers: { "Accept-Encoding": "gzip, deflate" } })
+})
+
+Cypress.Commands.add('goToCompareTwoCarsLink', () => {
+    cy.get(Selectors.compareTwoCars).click()
+})
+
+Cypress.Commands.add('checkComparePageIsOpen', () => {
+    cy.url().should('contain', testData.comparePage)
+    cy.get(Selectors.comparePageTitle).should('be.visible')
+})
+
+Cypress.Commands.add('addCar1Characteristics', (fileName) => {
+    cy.fixture(fileName).then((carInfo) => {
+        const makeValue = carInfo.make
+        const modelValue = carInfo.model
+        const yearValue = carInfo.year
+        cy.get(Selectors.addCar1MakeMenu).select(makeValue).then(() => {
+            cy.request({
+                method: 'GET',
+                url: testData.makeMenuEnpoint,
+            }).then((response) => {
+                expect(response.status).to.eq(200)
+            })
+        })
+        cy.get(Selectors.addCar1MakeMenu)
+            .should('have.value', makeValue.toLowerCase())
+            .then(() => {
+                cy.get(Selectors.addCar1ModelMenu).select(modelValue)
+                cy.get(Selectors.addCar1YearMenu).select(yearValue)
+            })
+    })
+})
+
+Cypress.Commands.add('addCar2Characteristics', (fileName) => {
+    cy.fixture(fileName).then((carInfo) => {
+        const makeValue = carInfo.make
+        const modelValue = carInfo.model
+        const yearValue = carInfo.year
+        cy.get(Selectors.addCar2MakeMenu).select(makeValue).then(() => {
+            cy.get(Selectors.addCar2ModelMenu).select(modelValue)
+            cy.get(Selectors.addCar2YearMenu).select(yearValue)
+        })
+    })
+})
+
+Cypress.Commands.add('clickComparisonButton', () => {
+    cy.get(Selectors.comparisonButton).click()
+})
+
+Cypress.Commands.add('checkCharacteristicsOnComparePage', (fileName) => {
```

Comment:
In case of expansion of this feature this approach might not be maintainable. First of all same case - pass object do not use file.

cy.get(Selectors.carComparisonPage).invoke('text').then((txt) => {
Object.Values(carInfo).forEach(objVal => expect(txt).to.contain(objVal))
})